giskard/ml_worker/websocket/listener.py

```diff
@@ -520,17 +520,20 @@ def run_test_suite(
     client: Optional[GiskardClient], params: websocket.TestSuiteParam, *args, **kwargs
 ) -> websocket.TestSuite:
     log_listener = LogListener()
+
+    loaded_artifacts = dict()
```

Comment:
Why is it initialized here if it's also set in the function with the 'if none' condition ?

Reply:
it' done to share the cache between the global arguments and test ones